sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/python/UserDefinedPythonDataSource.scala

```diff
@@ -300,6 +321,94 @@ private class UserDefinedPythonDataSourceRunner(
   }
 }
 
+case class PythonFilterPushdownResult(
+    dataSource: Array[Byte],
+    isFilterPushed: collection.Seq[Boolean])
```

Comment:
The name `isFilterPushed` looks confused. In fact, it contains both pushed and not pushed.

Reply:
`isFilterPushed(i)` means whether filter i is pushed. Maybe I can name it "isFilterPushedMask" to make it clearer.

Reply:
or add a comment to explain

Reply:
How about `filterPushedStatuses` ?